### PR TITLE
Rails 4 compatibility

### PR DESCRIPTION
**Pull request by kliuless**
Rails 3.2 is still supported

### CODE REVIEW

File: spec/active_record_check_spec.rb [hunk 1]
```diff
@@ -7,45 +7,80 @@
   class IsItWorking::TestActiveRecord < ActiveRecord::Base
   end
   
-  it "should succeed if the ActiveRecord connection is active" do
+  it "succeeds if the ActiveRecord connection is active" do
     connection = ActiveRecord::ConnectionAdapters::AbstractAdapter.new(double(:connection))
```
Comment (gdeoliveira) [inline spec/active_record_check_spec.rb]: Could we move `connection` to a `let`? It seems to be initialized the same way on several scenarios.

File: spec/active_record_check_spec.rb [hunk 2]
```diff
@@ -7,45 +7,80 @@
...
+        def k.name; to_s; end
+      end
+      klass_name = name.to_s.dup.freeze
+      klass.define_singleton_method(:to_s) { klass_name }
+      klass.establish_connection(adapter: 'sqlite3', database: ':memory:')
+      klass
+    end
+
+    it "succeeds on active connection" do
+      model_class = make_ar_klass
```
Comment (gdeoliveira) [inline spec/active_record_check_spec.rb]: Why are you assigning over the `model_class` obtained on line 56 here and in the following scenario?

Comment (kliuless) [inline spec/active_record_check_spec.rb]: Forgot to remove the assignments after creating the `let`. Fixed

File: spec/active_record_check_spec.rb (hunk 1 above)
Comment (kliuless) [inline spec/active_record_check_spec.rb]: Done

**Pull request merged**